ingesters/HttpIngester/config.go

```diff
@@ -254,6 +270,20 @@ func (c *cfgType) MaxBody() int {
 	return c.Max_Body
 }
 
+func (c *cfgType) GlobalTimestampWindow() (w timegrinder.TimestampWindow, err error) {
```

Comment:
these needs to check for negative durations, people will get confused and put those in.

Reply:
also move this into the IngestConfig so all ingesters can use it